scaffolds_actor_model/elixir/README.md

```diff
@@ -0,0 +1,27 @@
+# Actor Model
+
+This is an example of actors in Elixir. Since actors are built in Erlang/Elixir,
+this is simply an example of a way to use them.
+
+## Run
+
+```
+./mix run --no-halt
+```
+
+## Installation
+
+If [available in Hex](https://hex.pm/docs/publish), the package can be installed
+by adding `elixir` to your list of dependencies in `mix.exs`:
+
+```elixir
+def deps do
+  [
+    {:elixir, "~> 0.1.0"}
```

Comment:
Is that logical that elixir is version 0.1.0 in deps?

Reply:
Not at all. Thanks :)